doc/docs.md

```diff
@@ -388,34 +388,51 @@ println(s) // "hello\nworld"
 ```v
 a := 123
 ```
+
 This will assign the value of 123 to `a`. By default `a` will have the
 type `int`.
 
 You can also use hexadecimal notation for integer literals:
+
 ```v
 a := 0x7B
 ```
 ... or binary notation for integer literals:
+
 ```v
 a := 0b01111011
 ```
 ... or octal notation for specifying integer literals:
+
 ```v
 a := 0o173
 ```
 
 All of these will assign the same value 123 to `a`. `a` will have the
 type `int` no matter what notation you have used for the integer literal.
 
+V also supports writing numbers with `_` as separator:
+
+```v
+num := 100_000_0 // same as 1000000
+three := 0b0_11 // same as 0b11
+float_num := 312_2.55 // same as 3122.55
+hexa := 0x_FF // same as 255
+oct := 0o_173 // same as 0o173
```

Comment:
Same here.  May not be allowed as a leading character.  The `0x`, `0b`, etc. don't count.